src/components/label/label.visualroute.js

```diff
@@ -18,9 +18,11 @@ export const component = ({ themes }) => (
         Hello
       </Label>
     </Spec>
-    <ThemeProvider theme={themes.darkTheme}>
-      <Spec label="when inverted">
-        <Label tone="inverted">Hello</Label>
+    <ThemeProvider theme={{ ...themes.darkTheme, colorSurface: 'purple' }}>
```

Comment:
because the spec uses `colorSurface` as the background color, we need to use two themeproviders so we don't render black text on black background.